### PR TITLE
Combine `observe_new_views` into one in feedback crate

### PR DESCRIPTION
**Pull request by tcoratger**
Combine two `observe_new_views` calls into a single one in `init` function of feedback crate.

Release Notes:

- N/A

### CODE REVIEW

File: crates/feedback/src/feedback.rs [hunk 1]
```diff
@@ -20,43 +20,39 @@ actions!(
 );
 
 pub fn init(cx: &mut AppContext) {
-    // TODO: a way to combine these two into one?
-    cx.observe_new_views(feedback_modal::FeedbackModal::register)
-        .detach();
+    cx.observe_new_views(|workspace: &mut Workspace, cx| {
+        // First registration
```
Comment (maxdeviant) [inline crates/feedback/src/feedback.rs:24]: I don’t think we need this comment on the registration call (or the one below). They don’t provide any value.

File: crates/feedback/src/feedback.rs [hunk 2]
```diff
@@ -20,43 +20,39 @@ actions!(
...
+                cx.spawn(|_, _cx| async move {
+                    prompt.await.ok();
                 })
+                .detach();
+                cx.write_to_clipboard(ClipboardItem::new(specs.clone()));
+            })
             .register_action(|_, _: &RequestFeature, cx| {
-                let url = "https://github.com/zed-industries/zed/issues/new?assignees=&labels=enhancement%2Ctriage&template=0_feature_request.yml";
-                cx.open_url(url);
+                cx.open_url("https://github.com/zed-industries/zed/issues/new?assignees=&labels=enhancement%2Ctriage&template=0_feature_request.yml");
```
Comment (maxdeviant) [inline crates/feedback/src/feedback.rs:45]: I think we may have outlined these URL declarations to ensure `rustfmt` didn’t break due to the long line length.

Does `cargo fmt` still work for this file with them inlined?

File: crates/feedback/src/feedback.rs (hunk 1 above)
Comment (tcoratger) [reply crates/feedback/src/feedback.rs]: I've just removed

File: crates/feedback/src/feedback.rs (hunk 2 above)
Comment (tcoratger) [reply crates/feedback/src/feedback.rs]: Fixed

**Comment by maxdeviant** [comment]
Looks good! Thank you for the PR, @tcoratger!

**Pull request merged**